### PR TITLE
Add event listing

### PR DESCRIPTION
**Pull request by gorkaj**

### CODE REVIEW

**Comment by gorkaj** [comment]
s2
s1

File: src/front/ticketer/lib/backend_communication/logic/communication.dart [hunk 1]
```diff
@@ -56,6 +56,37 @@ class BackendCommunication {
...
+    if (!isInitialized) throw Exception("Not initilized");
+    Response response;
+    var options = dio.options;
+
+    try {
+      if (!token.isValid) {
+        throw ArgumentError("Token is not valid");
+      }
+      dio.options.headers.addAll({"Authorization": "Bearer ${token.token}"});
+      dio.options.contentType = Headers.formUrlEncodedContentType;
```
Comment (bjadczak) [inline src/front/ticketer/lib/backend_communication/logic/communication.dart:71]: please move to global settings

File: src/front/ticketer/lib/pages/common/event_tile.dart [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  Widget _getEventInfo(String property, String value, {TextStyle? style}) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Flexible(
+          flex: 2,
+          child: Text(
+            property,
+            style: const TextStyle(
+                fontWeight: FontWeight.w700, color: Colors.blue, fontSize: 16),
```
Comment (bjadczak) [inline src/front/ticketer/lib/pages/common/event_tile.dart:97]: we shouldn't use outright colors. This is like magic numbers when You change color scheme

File: src/front/ticketer/lib/pages/common/event_tile.dart [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            style: TextStyle(fontSize: 16, fontWeight: style?.fontWeight),
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _getContent() {
+    return SingleChildScrollView(
+      child: Column(
```
Comment (bjadczak) [inline src/front/ticketer/lib/pages/common/event_tile.dart:113]: Do You need Column here? Try something else this looks wrong to have column with one object

File: src/front/ticketer/lib/pages/user/user_landing_page.dart [hunk 4]
```diff
@@ -47,6 +70,38 @@ class _UserLandingPageState extends State<UserLandingPage> {
...
+            } else {
+              try {
+                _fetchMoreData();
+                setState(() {
+                  _pageNo++;
+                });
+              } catch (e) {
+                log(e.toString());
+              }
+              return const SizedBox(
```
Comment (bjadczak) [inline src/front/ticketer/lib/pages/user/user_landing_page.dart:90]: Please remove magic numbers

**Review by bjadczak — CHANGES_REQUESTED** [review]

**Review by bjadczak — DISMISSED** [review]

**Review by bjadczak — APPROVED** [review]

**Pull request merged**